crml/staking/src/rewards/mod.rs

```diff
@@ -63,6 +63,8 @@ pub trait Trait: frame_system::Trait {
 	type HistoricalPayoutEras: Get<u16>;
 	/// The reward payouts would be split among several blocks when their number exceeds this threshold.
 	type PayoutSplitThreshold: Get<u32>;
+	/// The number of staking eras in a fiscal era.
+	type FiscalDuration: Get<u32>;
```

Comment:
nit: `FiscalEraLength` / something with the unit

Reply:
Done